import-scripts/update_cfdna_clinical_sample_patient_ids_via_dremio.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+CLINICAL_SAMPLE_FILEPATH="/data/portal-cron/cbio-portal-data/cmo-access/mixed_MSK_cfDNA_RESEARCH_ACCESS/data_clinical_sample.txt"
+UPDATED_CLINICAL_SAMPLE_FILEPATH="data_clinical_sample_replaced.txt"
+PATIENT_ID_MAPPING_FILEPATH="patient_id_mapping.txt"
+PATIENT_ID_MAPPING_FILTERED_FILEPATH="patient_id_mapping_filtered.txt"
+PATIENT_ID_MAPPING_AMBIGUOUS_FILEPATH="patient_id_mapping_ambiguous.txt"
+TAB=$'\t'
+
+# filter patient id mappings of all rows missing either dmp-id or cmo-id
+if ! cat "$PATIENT_ID_MAPPING_FILEPATH" | egrep -v "^[[:space:]]" | egrep -v "[[:space:]]$" > "$PATIENT_ID_MAPPING_FILTERED_FILEPATH" ; then
+    echo "could not filter $PATIENT_ID_MAPPING_FILEPATH of rows missing a dmp id" >&2
+    exit 1
+fi
+
+# generate list of ambiguous cmo patient ids (have multiple associated dmp ids)
+if ! cat "$PATIENT_ID_MAPPING_FILTERED_FILEPATH" | sort | uniq | cut -f2 | sort | uniq -d > "$PATIENT_ID_MAPPING_AMBIGUOUS_FILEPATH" ; then
+    echo "could not findfilter $PATIENT_ID_MAPPING_FILEPATH of rows missing a dmp id" >&2
```

Comment:
This echo statement needs correcting.

Reply:
this was a simply error --- the error message was cloned but not updated. Good catch .. this has been fisxed